example/replication/app.js

```diff
@@ -6,8 +6,12 @@
 var loopback = require('../../');
 var app = loopback();
 var db = app.dataSource('db', { connector: loopback.Memory });
-var Color = app.model('color', { dataSource: 'db', options: { trackChanges: true }});
-var Color2 = app.model('color2', { dataSource: 'db', options: { trackChanges: true }});
+var Color = app.model(
+  app.registry.createModel('color', {}, { trackChanges: true }),
+  { dataSource: 'db' });
```

Comment:
Uff, there is too much stuff happening in a single statement. Please split this into two statements:

```js
var Color = app.registry.createModel('color', {}, { trackChanges: true });
app.model(Color, { dataSource: 'db' });
```